eox_hooks/receivers.py

```diff
@@ -0,0 +1,81 @@
+"""This module defines the signals receivers for defined hooks."""
+import logging
+from importlib import import_module
+
+from django.conf import settings
+
+from eox_hooks.constants import Status
+from eox_hooks.models import HookExecutionAudit
+
+log = logging.getLogger(__name__)
+
+DEFAULT_TASK_MODULE = "eox_hooks.tasks"
+DEFAULT_TASK_NAME = "default_task"
+
+
+def hooks_handler(sender, **kwargs):
+    """Eox-hooks signal receiver."""
+    if not getattr(settings, "USE_EOX_HOOKS", False):
+        return
+
+    try:
+        current_sender = sender.__name__
```

Comment:
do you have any example when the sender does not have` __name__` ?

Reply:
I'll add it